CHANGELOG.md

```diff
@@ -5,7 +5,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
 project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+- [**BREAKING**] Benchmark and `root` paths are now interpreted relative to the
+  location of the config file, instead of the current working directory.
+
+  For example, a config at path `<repo>/benchmarks/foo/tachometer.json` that
+  used to look as follows, and had to be run with `<repo>` as the cwd:
+
+  ```json
+  {
+    "root": ".",
+    "benchmarks": [
+      {
+        "url": "benchmarks/foo/index.html"
+      }
+    ]
+  }
+  ```
+
+  Should now look like this, and can be run from any cwd:
+
+  ```json
+  {
+    "root": "../..",
+    "benchmarks": [
+      {
+        "url": "foo.html"
```

Comment:
[no action needed]. Ooops I missed leaving this comment. Sorry! I realized the bottom example doesn't match the top one. It doesn't have to match, however should the url match in the second example - to show an easy way to migrate using the root?

This second example currently resolves to `<repo>/foo.html`.

Expected the second example to be:

```json
{
  "root": "../..",
  "benchmarks": [
      {
        "url": "benchmarks/foo/index.html"
      }
    ]
  }
```